sycl/doc/FAQ.md

```diff
@@ -2,51 +2,51 @@
 
 **Table of contents**
 
-1. [Developing with SYCL](#developing-with-sycl)
-1. [Using applications built with SYCL](#using-applications-built-with-sycl)
+1. [Developing with DPC++](#developing-with-dpc)
+1. [Using applications built with DPC++](#using-applications-built-with-dpc)
 1. [Common issues](#common-issues)
 1. [Device specific questions and issues](#device-specific-questions-and-issues)
 
 
-## Developing with SYCL
+## Developing with DPC++
 
-### Q: What do I need to start developing with SYCL?
-**A:** To get the full SYCL experience you need a SYCL-capable compiler. Intel
-SYCL compiler provides you with both host and device side compilation. Another
+### Q: What do I need to start developing with DPC++?
+**A:** To get the full DPC++ experience you need oneAPI DPC++ compiler. DPC++
```

Comment:
```suggestion
**A:** To get the full DPC++ experience you need the unique oneAPI DPC++ compiler. DPC++
```
It looks quite less generic this way :-)

Reply:
We are not so paranoid about our uniqueness :) After all everybody can fork our compiler and try something on top. Keep calling it DPC++ in that case might make sense since DPC++ extensions are there.